tests/security/attributes/Writer.cpp

```diff
@@ -102,6 +102,10 @@ Writer::svc()
     message.subject      = "Review";
     message.text         = "Worst. Movie. Ever.";
     message.count        = 0;
+    message.extra_space.length(args_.extra_space);
+    for (int i = 0; i < args_.extra_space; i++) {
+      message.extra_space[i] = 'a';
```

Comment:
Why don't we go ahead and make this a memset if we're going to set the values at all.

Reply:
Done.